### ISSUE TITLE
GPU was added to cart and ordered in the log, but on Amazon shows item currently unavailable.

### ISSUE THREAD

**Issue by hertzoglucas** [Hari-Nagarajan/fairgame#512]
**Describe the bug**

Here is the log:
2021-02-21 01:27:40,241|0.6.0|INFO|Found 1 offers for B08L8L9TCZ.  Evaluating offers...
2021-02-21 01:27:40,310|0.6.0|DEBUG|'check_stock' returned False
2021-02-21 01:27:43,311|0.6.0|DEBUG|Calling check_stock(<stores.amazon.Amazon object at 0x0000027847A53AF0>, 'B08P2HBBLX', 500.0, 750.0)
2021-02-21 01:27:43,672|0.6.0|DEBUG|Initial page title 
2021-02-21 01:27:43,678|0.6.0|DEBUG|        page url: https://www.amazon.com/dp/B08P2HBBLX/?pldnSite=1#aod
2021-02-21 01:27:47,263|0.6.0|DEBUG|After footer page title Amazon.com: ASUS Dual NVIDIA GeForce RTX 3060 Ti OC Edition Gaming Graphics Card (PCIe 4.0, 8GB GDDR6 Memory, HDMI 2.1, DisplayPort 1.4a, Axial-Tech Fan Design, Dual BIOS, Protective Backplate, GPU Tweak II): Computers & Accessories
2021-02-21 01:27:47,265|0.6.0|DEBUG|             page url: https://www.amazon.com/dp/B08P2HBBLX/?pldnSite=1#aod
2021-02-21 01:27:47,298|0.6.0|INFO|Found 1 offers for B08P2HBBLX.  Evaluating offers...
2021-02-21 01:27:47,334|0.6.0|INFO|Item in stock and in reserve range!
2021-02-21 01:27:47,335|0.6.0|INFO|Adding to cart
2021-02-21 01:27:47,342|0.6.0|INFO|Attempting Add To Cart with offer ID...
2021-02-21 01:27:49,843|0.6.0|DEBUG|'check_stock' returned True
2021-02-21 01:27:49,843|0.6.0|DEBUG|'run_asins' returned 'B08P2HBBLX'
2021-02-21 01:27:49,843|0.6.0|DEBUG|Calling navigate_pages(<stores.amazon.Amazon object at 0x0000027847A53AF0>, False)
2021-02-21 01:27:49,845|0.6.0|INFO|Navigating page title: 'Amazon.com Shopping Cart'
2021-02-21 01:27:49,846|0.6.0|DEBUG|Calling handle_cart(<stores.amazon.Amazon object at 0x0000027847A53AF0>)
2021-02-21 01:27:49,846|0.6.0|INFO|Looking for Proceed To Checkout button...
2021-02-21 01:27:50,424|0.6.0|INFO|Found Checkout Button
2021-02-21 01:27:50,555|0.6.0|INFO|Clicked Proceed to Checkout Button
2021-02-21 01:27:52,577|0.6.0|DEBUG|'handle_cart' returned None
2021-02-21 01:27:52,577|0.6.0|DEBUG|'navigate_pages' returned None
2021-02-21 01:27:52,578|0.6.0|DEBUG|Calling navigate_pages(<stores.amazon.Amazon object at 0x0000027847A53AF0>, False)
2021-02-21 01:27:52,579|0.6.0|INFO|Navigating page title: 'Amazon.com Checkout'
2021-02-21 01:27:52,579|0.6.0|DEBUG|Calling handle_checkout(<stores.amazon.Amazon object at 0x0000027847A53AF0>, False)
2021-02-21 01:27:52,820|0.6.0|INFO|Clicking Button  to place order
2021-02-21 01:27:55,848|0.6.0|DEBUG|'handle_checkout' returned None
2021-02-21 01:27:55,848|0.6.0|DEBUG|'navigate_pages' returned None
2021-02-21 01:27:55,848|0.6.0|DEBUG|Calling navigate_pages(<stores.amazon.Amazon object at 0x0000027847A53AF0>, False)
2021-02-21 01:27:55,849|0.6.0|INFO|Navigating page title: 'Amazon.com Checkout'
2021-02-21 01:27:55,849|0.6.0|DEBUG|Calling handle_checkout(<stores.amazon.Amazon object at 0x0000027847A53AF0>, False)
2021-02-21 01:27:55,872|0.6.0|INFO|Clicking Button  to place order
2021-02-21 01:27:57,001|0.6.0|INFO|Cleaning up after web driver...
2021-02-21 01:27:57,002|0.6.0|DEBUG|Killing 20456...
2021-02-21 01:27:57,002|0.6.0|DEBUG|Killing 17748...
2021-02-21 01:27:57,002|0.6.0|DEBUG|Killing 19912...
2021-02-21 01:27:57,003|0.6.0|DEBUG|Killing 15208...
2021-02-21 01:27:57,003|0.6.0|DEBUG|Killing 9332...
2021-02-21 01:27:57,003|0.6.0|DEBUG|Killing 7164...
2021-02-21 01:27:57,003|0.6.0|DEBUG|7164 not found. Continuing...
2021-02-21 01:27:57,003|0.6.0|DEBUG|Killing 18940...
2021-02-21 01:27:57,003|0.6.0|DEBUG|18940 not found. Continuing...
2021-02-21 01:27:57,003|0.6.0|DEBUG|Killing 16964...

Is this something I did wrong or too slow? 
Or issue with amazon?

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser  chrome
 - Version

**Comment by DakkJaniels** [comment]
If it had placed an order, it would have said "Order placed" after it clicked the button to place the order.  Something caused the program to stop running, but it's not clear from the log what that could be. No screenshots were taken I assume (in screenshots folder)?

**Comment by hertzoglucas** [comment]
Could it be that I was slower than another bot? or person lol?! 

I do have screenshots but they're are of no use because I had my browser minimized to a small window.

**Comment by DakkJaniels** [comment]
You are not the only other person running a bot, so yes, that is likely.

**Comment by hertzoglucas** [comment]
Okay, that's what I assumed initially. I have the browser big enough for useful ss if it happens again, ill report back if I can secure a card or that happens again.

**Comment by unapproachable** [comment]
The log shows what I believe to be a button with no text.  The code calls for the log message to include the text on the button, but the log here shows no text.  I'm wondering if that's a common occurrence or pointing to something odd on that page.  It does look like an unplanned exit of the bot.  Did you have --single-shot or only one ASIN group?  At least that would explain the shutdown after the attempted purchase.

**Comment by hertzoglucas** [comment]
No, --single-shot and I have 3 ASIN groups going. I did notice I received an email confirming a $0 order with no details and a link to "order details" which states, "This order has been cancelled" and "Continue shopping." 

I'm wondering is this issue arising from too many hits from my I.P. I am using a VPN as we speak to see if that is the issue.

**Comment by unapproachable** [comment]
Check the cancelled orders tab on Amazon to see what it was.  I had exactly that when they cancelled my TUF 3080.

![image](https://user-images.githubusercontent.com/74546317/109206832-5b819f80-7776-11eb-9d4b-b69165def631.png)

**Comment by YangDaGod** [comment]
> No, --single-shot and I have 3 ASIN groups going. I did notice I received an email confirming a $0 order with no details and a link to "order details" which states, "This order has been cancelled" and "Continue shopping."
> 
> I'm wondering is this issue arising from too many hits from my I.P. I am using a VPN as we speak to see if that is the issue.

any success? do you know if it was too many hits from your I.P. that caused the order to cancel?
